public/js/actions/index.js

```diff
@@ -0,0 +1,123 @@
+import chartList from 'utils/chartList';
+import api from 'services/Api';
+import { reEstablishSession } from 'panda-session';
+
+/*-------- CHARTS --------*/
```

Comment:
Overall, I feel this file is a bit long and could be split up, just to make it easier to read